client/src/components/MealForm.js

```diff
@@ -29,17 +30,19 @@ const useStyles = makeStyles((theme) => ({
 
 function MealForm(props) {
     const classes = useStyles();
-    const [picURL, setMealPicUrl] = React.useState("");
+    const [picURL, setMealPicUrl] = React.useState(
+        props.meal.picURL ? props.meal.picURL : ""
+    );
 
     const meal = props.meal;
     const price = meal.price !== "" ? meal.price / 100 : meal.price;
     const cuisineType = meal.cuisineType[0] ? meal.cuisineType[0] : "";
     const pic = meal.picURL !== undefined ? meal.picURL : "";
+    const user = React.useContext(UserContext);
 
     const validationSchema = Yup.object().shape({
         title: Yup.string().required("Required!"),
-        price: Yup.number()
-            .typeError("Price must be a number")
+        price: Yup.string()
```

Comment:
Would $5.1 pass this regex test? I'm wondering if we want `{2}` instead of `{0,2}`.

Reply:
Yes, it does pass.  From my understanding the {0,2} makes it a range so that it will pass if its 5.1 while if its {2} it will only pass if its 5.10.  I think its ok if the number is 5.1 So maybe I should change the wording of  error message

Reply:
Ok. As long as that's intentional it's fine. I think we can keep the error message wording